go.mod

```diff
@@ -101,4 +101,6 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
+replace github.com/codeready-toolchain/toolchain-common => github.com/xcoulon/toolchain-common v0.0.0-20221109080512-912af81e4fa2
```

Comment:
Friendly reminder

Reply:
Just curious, since there is no reference of the toolchain-common change, is it there any other reason apart from good convention to include this replace?

Reply:
correct, there was no reason, except to keep member-operator up-to-date with toolchain-common and make sure nothing is broken with this module upgrade